test/postgresDbTest.js

```diff
@@ -11,18 +10,6 @@ const id = Math.floor(Math.random() * 300000);
 const queryById = `select * from movies where id = ${id}`;
 
 describe('Movies', () => {
-  // it('testing', (done) => {
```

Comment:
If you don't need this test at the moment, you can just x it out. It's cleaner/easier than commenting the whole block out.

Reply:
This has already been deleted.